json.html.markdown

```diff
@@ -64,4 +64,30 @@ Drawbacks of JSON include lack of type definition and some sort of DTD.
 
   "that was short": "And, you're done. You now know everything JSON has to offer."
 }
+
+// Object JSON FROM String
+// The JSON.parse() method parses a string as JSON, optionally transforming the value produced by parsing.
+
+// Sintax:
+// JSON.parse(text[, reviver])
```

Comment:
@vendethiel This is github's markdown parser interpreting `[, reviver]` as bad JSON. JSON doesn't have comments, and the `[, reviver]` is interpreted as an array literal, but since `,` is not a valid first token inside an array literal, it borks the parsing. This entire comment block would be best be separated from the json block.